packages/react-query/src/__tests__/useQueries.test-d.tsx

```diff
@@ -104,7 +105,7 @@ describe('UseQueries config object overload', () => {
       type Data = string
 
       const useCustomQueries = (
-        options?: Omit<UseQueryOptions<Data>, 'queryKey' | 'queryFn'>,
+        options?: OmitKeyOf<UseQueryOptions<Data>, 'queryKey' | 'queryFn'>,
```

Comment:
We don't have to worry our misspell like below

```ts
type NoNecessaryError = Omit<UseQueryOptions<Data>, 'queryTey' | 'qeuryFn'>,
```

queryTey, qeuryFn(wrong spell) make type error if with OmitKeyOf

```ts
type NecessaryError = OmitKeyOf<UseQueryOptions<Data>, 'queryTey' | 'qeuryFn'>, // will make type error
```